.circleci/config.yml

```diff
@@ -40,7 +45,30 @@ jobs:
       - store_artifacts:
           path: coverage/lcov-report/
 
+      - run: npm run test:performance:ci
+
+      - store_artifacts:
+          path: performance-tests/__snapshots__
```

Comment:
while not totally required, this does give us a historical view within a branch (since when we make our `-artifacts` branch, we always just force-push over the commit)